lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -117,7 +127,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $output->writeln(sprintf('Generated new migration class to "<info>%s</info>" from schema differences.', $path));
     }
 
-    private function buildCodeFromSql(Configuration $configuration, InputInterface $input, array $sql)
+    private function buildCodeFromSql(Configuration $configuration, array $sql, $formatted=false, $lineLength=0)
```

Comment:
yeah, or 120 to be consistent with the default value of the CLI option itself :)